hbase-server/src/test/java/org/apache/hadoop/hbase/master/MockRegionServer.java

```diff
@@ -205,10 +205,11 @@ int getThenIncrement() {
   }
 
   /**
-   * Use this method filling the backing data source used by {@link #get(RpcController, GetRequest)}
-   * @param regionName
-   * @param row
-   * @param r
```

Comment:
In future, could just purge params that do not have description.

Reply:
You mean inside test code, right?